backend/db/ddl-14-concepts_with_counts_ungrouped.jinja.sql

```diff
@@ -17,5 +17,5 @@ SELECT DISTINCT
 FROM {{schema}}concept c
 LEFT JOIN {{schema}}deidentified_term_usage_by_domain_clamped tu ON c.concept_id = tu.concept_id);
 
-CREATE INDEX ccu_idx1{{optional_index_suffix}} ON {{schema}}concepts_with_counts_ungrouped(concept_id);
+CREATE INDEX ccu_idx1{{optional_index_suffix}} ON {{schema}}concepts_with_counts_ungrouped{{optional_suffix}}(concept_id);
```

Comment:
Had to update some DDL files with `{{optional_suffix}}` to get working w/ the refreshing.